crates/tabby-common/src/api/event.rs

```diff
@@ -3,14 +3,18 @@ use utoipa::ToSchema;
 
 #[derive(Serialize, Deserialize, ToSchema, Clone, Debug)]
 pub struct LogEventRequest {
-    /// Event type, should be `view` or `select`.
+    /// Event type, should be `view`, `select` or `dismiss`.
```

Comment:
Maybe we should use an enum to model this, or `Discriminant<Event>`? I believe I forgot to modify this comment when adding the `Completion` variant - using `Discriminant<Event>` would make it clear through the type system what this is, and not require updating the comment every time a new event is added.

Reply:
there's no need to modify this, completion event are not logged through this route.